beheer.md

```diff
@@ -104,13 +104,13 @@ docker service ps getstartedlab_web | Haal de gegevens op van alle tasks die de
 docker stack rm getstartedlab | Afsluiten van de applicatie.
 docker swarm leave --force | Afsluiten van de swarm.
 
-**Criteria**
-
-* API specificatie (OAS3) moet beschikbaar zijn en uit de imvertor genereerbaar zijn
-* Documentatie (zowel technisch als functioneel) moet beschikbaar zijn
-* De referentie-implementatie moet gereed en up-to date zijn met dan geldende OAS3
-* Het gebruikte Informatiemodel (minimaal UML)
-* Geautomatiseerde test en buildstraat (CI) moet beschikbaar voor de RI
-* De standaard moet ingebouwd zijn bij leveranciers (3 of meerdere leveranciers)
-* De standaard bevat een tabel waarin wordt aangegeven welke API versies met welke andere API versies compatible zijn.
+##Criteria##
+
+- API specificatie (OAS3) moet beschikbaar zijn en uit de imvertor genereerbaar zijn. Dit betekent dat zowel de Semantische Informatiemodellen (SIM), Uitwisselings Gegevensmodellen (UGM) als het Bericht Specificatiemodel (BSM) in UML beschikbaar moeten zijn.
+- Documentatie (zowel technisch als functioneel) moet beschikbaar zijn
+- De referentie-implementatie moet gereed en up-to date zijn met dan geldende OAS3
+- Het gebruikte Informatiemodel (minimaal UML)
+- Geautomatiseerde test en buildstraat (CI) moet beschikbaar voor de RI
```

Comment:
Wat is beschikbaar?

Reply:
In de demo van 31 januari van ZGW werd dit component genoemd. Om die reden heb ik hem hier opgenomen.

Reply:
Het moet er m.i. ook wel zijn maar wat houdt beschikbaar in?

Reply:
> Het moet er m.i. ook wel zijn maar wat houdt beschikbaar in?

We moeten in staat zijn om op elk moment dat we dat willen de geautomatiseerde testen te draaien. Dat hoeft niet 7 x 24 uur, deze hoeven niet elke dag automatisch uitgevoerd te worden en als we dat lokaal kunnen is het mogelijk ook al voldoende.
We moeten dus een omgeving hebben waarin we dit kunnen doen en alle scenario's die getest moeten worden moeten er dus ook zijn.

Als ik deze beschrijving hieraan toevoeg is het dan duidelijk genoeg of niet?